samples/bluetooth/hci_spi/prj.conf

```diff
@@ -1,10 +1,6 @@
 CONFIG_GPIO=y
 CONFIG_SPI=y
 CONFIG_SPI_SLAVE=y
-CONFIG_MAIN_STACK_SIZE=512
-CONFIG_BT=y
-CONFIG_BT_HCI_RAW=y
 CONFIG_BT_MAX_CONN=20
```

Comment:
may consider using default of 16 if no specific reason for this

Reply:
Could just go to default IMO.